scripts/kubectl-kubestellar-ensure-location

```diff
@@ -108,13 +108,25 @@ fi
 
 set -e
 
+echo "--- current directory is $PWD"
+cwsi=$(kubectl ws "${kubectl_flags[@]}" .)
+cwsi=${cwsi#*'"'}; cwsi=${cwsi%'"'*}
```

Comment:
Isn't there a `--short` flag that makes this unnecessary?

Reply:
Is `cwsi` only used in `echo "current cwsi is $cwsi"`?  If so then wouldn't it be simpler to do `kubectl ws "${kubectl_flags[@]}" .` there and delete all the messing with `cwsi`?

Reply:
I don't know. Will research.